.github/workflows/ci.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8"]
+        django-version: ["pinned", "4.2"]
```

Comment:
We've implemented `pinned` logic only in edx-platform to make sure we are testing with the same patch version as production. For other IDAs we don't need such strict testing so you don't need to copy the method from edx-platform here. 
Adding `django42` in toxenv below should suffice for the scope of this repo.

Reply:
I don't think so we need to keep only django42 as we have decided to remove `toxenv` @UsamaSadiq